src/core/validators/validators.ts

```diff
@@ -15,18 +15,20 @@ export type ValidatorFn = (
  * @param value Value to test
  */
 const Required: ValidatorFn = (value) => {
-  if (!value) {
+  if (!value || (value?.trim?.() === '')) {
     return Promise.resolve({ required: true });
   }
   return Promise.resolve(null);
 };
 
 /**
+ * DEPRECATED - Please use the Required validator instead
```

Comment:
Can you please use the '@deprecated <some text>' JSDoc tag
(See https://jsdoc.app/tags-deprecated.html)

Reply:
Thank you for the suggestion :D